doc/nrf/ug_bt_qualification.rst

```diff
@@ -0,0 +1,109 @@
+
+By using pre-qualified designs, you can omit the testing phase during your product's qualification.
+
+The following product types are used and qualified by Nordic Semiconductor:
+
+* End product
+* Controller Subsystem product
+* Host Subsystem product
+* Profile Subsystem product
+* Component product
+
+All versions of Controller Subsystems (:ref:`ug_ble_controller_softdevice` versions), Host Subsystems and Components created by Nordic Semiconductor are qualified by them by default.
+Some Profile Subsystems (e.g., :ref:`ug_bt_mesh` versions) are qualified as well.
+If you want to use a Profile Subsystem that is not on the `Bluetooth Qualification Listing`_, request the relevant product's qualification by Nordic Semiconductor.
+
+
+Qualification process
+*********************
+
+The product qualification cannot be inherited, hence you must complete this process by yourself.
+You can either perform a full qualification process for a new product, or edit an existing entry for a product that has been already qualified.
+
+New entry creation
+==================
+
+The only way to qualify your product is to complete the `Bluetooth SIG's Qualification Process`_ under your member company's account.
+To check if your company is already a SIG member, go to `Join Bluetooth SIG`_ page and search the Member Directory list.
+
+Log into `Launch Studio`_, the official qualification tool, that will guide you through one of the following paths:
+
+* `Qualification process with no required testing`_ - If your complete product uses only pre-qualified End products or Subsystem products.
+* `Qualification process with required testing`_ - If your complete product uses a new Bluetooth design or pre-qualified Component products.
+
+.. note::
+   A *design* is a specific configuration of hardware or software implementation of adopted Bluetooth specifications.
+
+
+Existing entry update
+=====================
+
+Update an existing qualification entry through `Launch Studio`_ in the following cases:
+
+* You changed an existing product's design significantly so that the testing is required.
+* Your product can be added as a new model to one of your existing qualifications.
+* You switched to another SDK version for an existing product.
+
+If you want to switch to another SDK version for an existing product, perform the following steps:
+
+1. Go to the `Nordic Semiconductor Infocenter`_, search for the SoC or SiP model you use, and navigate to the relevant Compatibility Matrix directory.
+
+#. Open the Bluetooth QDIDs article and, based on the table, confirm the following information:
+
+  a. That relevant Host and SoftDevice Controller Subsystems implemented in the SDK version you want to use are pre-qualified.
+     If yes, perform one of the following actions:
+
+     * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones used for your product, then no further action is required.
```

Comment:
```suggestion
     * If the version you want to use is fully pre-qualified, and has the same Qualification Design ID (QDID) numbers as the ones currently used for your product, then no further action is required.
```
Adding this suggestion as during the first read it wasn't entirely clear for me which QDIDs to compare.

Reply:
Fair point, corrected.